### PR TITLE
[RS-173] update documentation

### PR DESCRIPTION
**Pull request by Zyruks**
## Description
:memo: add contributing file

The best feature of this contribute file is that it provides a comprehensive step-by-step guide on how to contribute to the project, including finding or reporting issues, forking the project, creating a new branch, and making changes to the codebase. It also includes links to the project's README and code of conduct, which provides further information and guidelines on how to contribute to the project. Additionally, the file provides useful code snippets and examples to help contributors understand the process better. 

Some of the key features of this contributing file are:

- Incorporate a GIF to provide a visual representation and enhance the user experience.
- Include emojis strategically to add more visual appeal and break up text blocks.
- Provide clear, step-by-step instructions that are easy to follow, especially for newcomers to GitHub and Git, to help them understand how to contribute effectively.

## Related Issue
#173 

## Motivation and Context

The motivation for creating a contributing file is to make it easier for people to contribute to your project. When starting out with a new project, it can be intimidating to figure out how to get involved and where to start. A well-written contributing file can provide guidance and instructions for potential contributors, making it easier for them to understand how they can help and what types of contributions are welcome.

Additionally, a contributing file can help maintain a standard of quality and consistency for the project. By outlining specific requirements and guidelines for contributions, it ensures that all contributions are aligned with the project's goals and meet a certain standard of quality. This can help maintain the overall health and sustainability of the project over time.

## Types of changes

- [X] Docs change / refactoring / dependency upgrade
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

- [X] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by Zyruks** [comment]
## Test
If you want to check in deep this contributing file, you can go to [test repo](https://github.com/Zyruks/zyruks-test/blob/main/.github/CONTRIBUTING.md)

## Visual:

image


image

image

image

image

**Comment by Zyruks** [comment]
# Improvements 

Move the CONTRIBUTING.md file to docs
add CODE_OF_CONDUCT.md
add SECURITY.md
add SUPPORT.md
add Spanish version for every file


Spanish version need to be re-check because I make a quick translation

English version have a way to enter the Spanish version and vice versa.

### Example:
image

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+- [ ] New feature
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+```
+
+</details>
+<details>
+<summary style="color: #6366F1; font-weight:bold">Maintaining or improving the quality of the project:</summary>
+
+```
+- [ ] My code follows the code style this project.
```
Comment (serudda) [inline docs/CONTRIBUTING.md:218]: CReo que estos checks sobran, la verdad no creo que sirvan para mucho.

File: docs/ES/CONTRIBUTING_ES.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+```
+
+</details>
+
+<details>
+<summary style="color: #6366F1; font-weight:bold">Mantener o mejorar la calidad del proyecto:
+</summary>
+
+```
+- [ ] My code follows the code style this project.
```
Comment (serudda) [inline docs/ES/CONTRIBUTING_ES.md:219]: Aqui lo mismo la verdad no creo que esto agregue nada... la gente va a poner cualquier cosa

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (Zyruks) [reply docs/CONTRIBUTING.md:218]: Si pensaba modificarlo cuando cambiáramos el template del PR, pero lo podemos quitar.

**Comment by Zyruks** [comment]
Make the changes from the review and update broken links.

**Pull request merged**